evms/evmone.go

```diff
@@ -54,11 +54,17 @@ func (evm *EvmoneVM) Name() string {
 }
 
 func (evm *EvmoneVM) GetStateRoot(path string) (root, command string, err error) {
-	cmd := exec.Command(evm.path, "--ignore-state-root", "--ignore-logs", "--trace", path)
+	cmd := exec.Command(evm.path, "--trace-summary", path)
 	data, err := cmd.CombinedOutput()
```

Comment:
Looking at the reference outputs -- we're only using the stderr aren't we? In that case, no need to use `CombinedOutput()`. But there's no easy `StderrOutput()` handy, so this is fine for now